lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -141,6 +140,21 @@ class window.Turbolinks
 
     return
 
+  getNodesMatchingRefreshKeys = (keys) ->
+    matchingNodes = []
+    for key in keys
+      for node in document.querySelectorAll("[refresh=#{key}]")
+        matchingNodes.push(node)
```

Comment:
yeah, previously was finding them all & replacing, now just finding them and returning them (smaller methods ftw)